frontend/src/app/shared/components/op-view-select/op-view-select.component.ts

```diff
@@ -121,10 +121,7 @@ export class ViewSelectComponent extends UntilDestroyedMixin implements OnInit {
       .onActivate(...this.menuItems)
       .subscribe(() => this.initializeAutocomplete());
 
-    this.views$ = combineLatest(
```

Comment:
We usually use the one-per-line style for two reasons: When making changes to the array it is better visible which observables have been removed/added. This also reduces possible merge conflicts later. Further, I think for those who work in a vim editor, it easier to grep those elements. Although, Ben is probably better in explain the concrete reasons for that last one 😅 
So simply adding the missing array brackets should be sufficient to make it look like this: 
```
this.views$ = combineLatest([
  this.search$,
  this.viewCategories$,
])
```

Reply:
Yes, good idea 👍

Reply:
I fixed the lines by hand. As I didn't got it to work automatically in eslint or edit config.